zerver/webhooks/github/tests.py

```diff
@@ -37,6 +37,10 @@ def test_push_delete_branch(self) -> None:
         expected_message = "eeshangarg [deleted](https://github.com/eeshangarg/public-repo/compare/2e8cf535fb38...000000000000) the branch feature."
         self.check_webhook("push__delete_branch", "public-repo / feature", expected_message)
 
+    def test_push_force(self) -> None:
+        expected_message = "sbansal1999 [pushed](https://github.com/sbansal1999/testing-gh/compare/9edbee9dc904...c0dcba5885f8) 2 commits to branch fixing-typo.\n\n* d ([70029144adc](https://github.com/sbansal1999/testing-gh/commit/70029144adc043fbe6f50498ab36acd3c4ba302f))\n* e ([c0dcba5885f](https://github.com/sbansal1999/testing-gh/commit/c0dcba5885f8d9def1f798e3f5eb24e0ace5a030))"
+        self.check_webhook("push__force", "testing-gh / fixing-typo", expected_message)
```

Comment:
Hmm, this is a force push, but not the typical example of one - say removing the last commit from a branch, where there wouldn't be any new ones. Can you add another fixture for that?

Reply:
You are right, adding a fixture that looks more like a regular force push will be better a idea. Thanks for the suggestion.